docs/docs/monitoring/analytics/realtime-markers.mdx

```diff
@@ -0,0 +1,87 @@
+
+<RasaProLabel />
+
+:::info Available in Rasa Pro from 3.6.0
+:::
+
+<RasaProBanner />
+
+In Rasa Pro, [Markers](../../markers.mdx) is a powerful feature that allows
+you to track and extract custom conversational events. With the Analytics
+Data Pipeline, you can now process these Markers in real-time, enabling
+you to gain valuable insights and enhance the performance of your Rasa
+Assistant. In this guide, we'll explore how to leverage real-time
+analysis of Markers to track solution rate and abandonment in your conversations.
+
+## Defining Markers
+
+Please consult the [Markers](../../markers.mdx/#defining-markers) section of Rasa documentation
+for details about defining Markers.
+
+## Enable Real-time Processing
+
+To enable real-time analysis of Markers in your Rasa project, you need to
+upload the markers to Analytics Data Pipeline with the `rasa markers upload`
+command. This command uploads the marker configuration YAML and starts the
+processing of Markers in the Analytics Data Pipeline. To get started,
+make sure you have the latest version of Rasa installed. Then, open your command line
+interface and navigate to your Rasa project directory. Run the following command:
+
+```
+rasa markers upload --config=<path-to-config-file> -d=<path-to-domain-file> -rasa-pro-services-url=<url>
+```
+
+By default, this command validates the marker configuration file against
+your bot's domain.yml file. To specify a different domain file, use the
+optional `-d` argument.
+
+This command should be run whenever there is a change in the marker
+configuration file. The changes might include addition of new markers,
+changing an existing marker or removing an existing marker.
+
+:::note
+
+This command **uploads** the marker configurations to the data pipeline. The pipeline assumes the configuration
+file is the source of truth and only processes the markers defined in it. If you remove a marker from this file,
+and run this command, then the processing for that marker is stopped.
+
+:::
+
+### Configuring the CLI command
+
+Visit our [CLI page](./command-line-interface.mdx#rasa-marker-upload)
+for more information about the command line arguments available.
+
+## How are Markers processed?
+
+Once the marker definitions are uploaded to the Analytics Data Pipeline, they
+are stored in the `rasa_patterns` table. As the Kafka Consumer starts receiving events
+from the Rasa Assistant, it starts analyzing them for markers. The Pipeline 
+processes all the events from the Kafka Event Broker and identifies points of 
+interests in the conversation that matches the marker. The extracted markers are stored
+in the data warehouse.
+
+The evaluation of Markers in the Pipeline is similar to the `rasa evaluate markers`
+command which can be used to process markers from the conversations in Tracker Store.
+Read more about it [here](../../markers.mdx/#extracting-markers)
+
+Extracted markers are added to the `rasa_markers` table in the data warehouse
```

Comment:
Can you add an example query and graph for markers on the Example queries page?

Reply:
I can add an example query and graph to visualise `num_preceeding_user_turns` for a marker that tracks when user has completed the form. It'll visualise the number of turns users took to complete the form and can help give insight if the users are struggling to complete the form. Will something like that be helpful? It's only one of the many use cases of this feature

Reply:
A form related example would be really good. When Alana and I recently wrote the blog post on this subject, we really needed a completion rate example. For a given form or story, how many people started the form and how many completed it successfully.

Reply:
> For a given form or story, how many people started the form and how many completed it successfully.

I'll add this example to this documentation

Reply:
@rgstephens With a release schedule of next week, we're short on time to add this example at the moment but I've created a ticket at https://rasahq.atlassian.net/browse/ATO-1227 for this and we'll add this example in a micro release